code/game/objects/effects/poster_motivational.dm

```diff
@@ -152,6 +152,14 @@
 	When people read this poster they'll feel better!"
 	icon_state = "bike"
 
+/obj/structure/sign/poster/quirk/crew/mining
```

Comment:
Why is this a quirk poster? This isn't exactly an inspirational message.

Reply:
It's also not in the same colourscheme or "hand-drawn visual" style of the other quirk posters, given its cinematic poster design it's hard to imagine someone made this with some pencils or whatever.